File: 10 - Python/Exercícios/03 - Estruturas Repetitivas/For/Exercício 05.py
#Exercício - Par Ímpar

n = int(input("Quantos números você vai digitar? "))

for i in range(n):
    
    numero = int(input("Digite um número: "))

    if numero == 0:
        print("Nulo")
    elif numero % 2 == 0:
        if numero > 0:
            print("Par positivo")
        else:
            print("Par negativo")
    else:
        if numero > 0:
            print("Ímpar positivo")
        else:
            print("Ímpar negativo")
    print("")

File: 07 - C++/Exercícios/03 - Estruturas Repetitivas/For/Exercício 05.cpp
//Exercício - Par ímpar

#include <iostream>

using namespace std;

int main(){

    int i, N, x;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    for(i = 1; i <= N; i++){

        cout << "Digite um numero: ";
        cin >> x;

        if(x == 0){
            cout << "Nulo\n" << endl;
        }
        else if(x % 2 == 0){
            if(x > 0){
                cout << "Par positivo\n" << endl;
            }
            else{
                cout << "Par negativo\n" << endl;
            }
        }
        else{
            if(x > 0){
                cout << "Impar positivo\n" << endl;
            }
            else{
                cout << "Impar negativo\n" << endl;
            }
        }
    }

    return 0;

}